backend/b2/b2.go

```diff
@@ -160,7 +160,19 @@ free egress for data downloaded through the Cloudflare network.
 Rclone works with private buckets by sending an "Authorization" header.
 If the custom endpoint rewrites the requests for authentication,
 e.g., in Cloudflare Workers, this header needs to be handled properly.
-Leave blank if you want to use the endpoint provided by Backblaze.`,
+Leave blank if you want to use the endpoint provided by Backblaze.
+
+The URL provided here SHOULD have the protocol and SHOULD NOT have
+a trailing slash or specify the /file/bucket subpath rclone will
```

Comment:
Full stop after `subpath` maybe?

Reply:
I went with "as" instead, thought it gives a bit more of continuity to the sentence.

What do you think?